### PR TITLE
Feature/dockerize redis

### PR DESCRIPTION
**Pull request by crisanlucid**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+# sbt
+ENV SBT_URL=https://dl.bintray.com/sbt/native-packages/sbt
+ENV SBT_VERSION 0.13.15
+ENV INSTALL_DIR /usr/local
+ENV SBT_HOME /usr/local/sbt
+ENV PATH ${PATH}:${SBT_HOME}/bin
+
+COPY svc /svc
+#EXPOSE 9000 9443
```
Comment (cricanr) [inline Dockerfile]: Please cleanup, remove commented out code or enable if we really need it only. 
The same for all following lines in this file.

File: README.md [hunk 2]
```diff
@@ -13,37 +13,116 @@ TODO
 How to run it
 -------
 Prerequisites
+
 1) Start dependencies (downstream services) first:
+
+Linux:
```
Comment (cricanr) [inline README.md:21]: Maybe later we can make another ticket with Mac instructions :) I could do that. What do you think?

File: app/stream/RedisStoreClient.scala [hunk 3]
```diff
@@ -17,9 +17,18 @@ trait IStoreClient {
...
+// TODO: Add back DI RedisClient, need to configure to use RedisClient with custom constructor (IP, PORT)
+// FOR now using new operator to try it works
+class RedisStoreClient @Inject()
                                 (implicit configuration: Configuration)
   extends IStoreClient {
+  private def getRedisIpParam=  {
+    Option(System.getProperty("redis.ip")).getOrElse("172.28.0.2")
+  }
+
+  println(s"param: ${System.getProperty("redis.ip")}")
```
Comment (cricanr) [inline app/stream/RedisStoreClient.scala]: This needs to go away.

File: build.sbt [hunk 4]
```diff
@@ -21,3 +27,7 @@ libraryDependencies ++= Seq( jdbc , ehcache , ws , specs2 % Test , guice,
 )
 
 unmanagedResourceDirectories in Test <+=  baseDirectory ( _ /"target/web/public/test" )
+
+
```
Comment (cricanr) [inline build.sbt]: remove commented out code

File: project/plugins.sbt [hunk 5]
```diff
@@ -2,4 +2,6 @@ logLevel := Level.Warn
 
 resolvers += "Typesafe repository" at "http://repo.typesafe.com/typesafe/releases/"
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.6.16")
\ No newline at end of file
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.6.16")
+addSbtPlugin("com.tapad" % "sbt-docker-compose" % "1.0.34")
+//addSbtPlugin("com.typesafe.play" % "sbt-native-packager" % "2.6.16")
```
Comment (cricanr) [inline project/plugins.sbt]: remove commented out code

**Review by cricanr — CHANGES_REQUESTED** [review]
We need to fix the points specified in the review and to get docker compose up of the project to work correctly.

File: README.md (hunk 2 above)
Comment (crisanlucid) [reply README.md:21]: yes; will be CrossPlatform

File: app/stream/RedisStoreClient.scala (hunk 3 above)
Comment (crisanlucid) [reply app/stream/RedisStoreClient.scala]: with the new PULL ; will be ok ?

File: project/plugins.sbt (hunk 5 above)
Comment (crisanlucid) [reply project/plugins.sbt]: here suppose to remove the plugins

File: app/stream/RedisStoreClient.scala (hunk 3 above)
Comment (cricanr) [reply app/stream/RedisStoreClient.scala]: yes, this is obsolete.

File: docker-compose.yml [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+#      - RHOST=redis
+#    networks:
+#      - redis-net
+#    depends_on:
+#      - redis
+#      - ratingsstream
+#    ports:
+#      - 9000:9000
+#      - 9443:9443
+#    command: /svc/bin/start -Dhttps.port=9443 -Dredis.ip="$REDIS_IP" -Dplay.crypto.secret=secret
```
Comment (crisanlucid) [inline docker-compose.yml:19]: how do you start the server HTTPS from play framework ?

File: Dockerfile [hunk 7]
```diff
@@ -0,0 +1,11 @@
+FROM openjdk:8-jre
+
+# sbt
+ENV SBT_URL=https://dl.bintray.com/sbt/native-packages/sbt
+ENV SBT_VERSION 0.13.15
+ENV INSTALL_DIR /usr/local
+ENV SBT_HOME /usr/local/sbt
+ENV PATH ${PATH}:${SBT_HOME}/bin
+ENV REDIS_IP=${REDIS_IP}
```
Comment (crisanlucid) [inline Dockerfile:9]: how will take the ip REDIS ? 
suppose to be a command but everything is commented ?

File: docker-compose.yml (hunk 6 above)
Comment (cricanr) [reply docker-compose.yml:19]: I took out Play for now from docker-compose. We will start it separately. This is temporary.

Comment (crisanlucid) [reply docker-compose.yml:19]: k

**Review by cricanr — APPROVED** [review]

**Pull request merged**